DeviceIndependentQKD/utils.py

```diff
@@ -0,0 +1,39 @@
+#Define protocol parameters and utility functions
+import math
+import numpy as np
+gamma = 0.2                     #Fraction of test rounds
+w_exp = (2+math.sqrt(2))/ 4     #Expected winning probability of CHSH game in an honest implentations
+del_est = 1e-3                  #Width of statistical interval for the Bell test
```

Comment:
Please  leave a comment about how this parameter should be set

Reply:
More information about the parameter added